org.agileware.natural.cucumber.feature/feature.xml

```diff
@@ -2,57 +2,9 @@
 <feature
       id="org.agileware.natural.cucumber.feature"
       label="Cucumber Editor"
-      version="0.7.6.qualifier"
+      version="1.4.0.qualifier"
```

Comment:
just a curiosity: why 1.4?

Reply:
It was 1.3.5.qualifier in MANIFEST, I think I went with the wrong version number. The hamcrest jars should be removed. For some reason I was having trouble getting Eclipse to recognize the classes. I probably need to do some kind of clean build.

At any rate, I've reset my master branch and am attempting to do some minor incremental updates starting with Eclipse Mars, and progressing from there. Sorry for the premature PR, I had meant to follow up with a note that this was just for comparison purposes only, so it would be easier to see where I've made changes.